### DISCUSSION TITLE
Admin password reset

### DISCUSSION THREAD

**Discussion by CharletsNfts** [uvdesk/community-skeleton#795] [Q&A] [answered]
How does one rests admin password if forgotten, I have forgotten my password now I can't log in? May you please help me

**Comment by papnoisanjeev** [comment, marked as answer]
@CharletsNfts 
**Option 1:** 
You can reset password from **forgot password** option only if you have enabled your **swiftmailer** before.

**Option 2:** 
You need to change from **uv_user** table  **password** column in your **database**.
Replace existing encypted password with **BFEQkknI/c+Nd7BaG7AaiyTfUFby/pkMHy3UsYqKqDcmvHoPRX/ame9TnVuOV2GrBH0JK9g4koW+CgTYI9mK+w==**

Now you can login using a default Password: **password**

You can change password after login into your profile.
